### DISCUSSION TITLE
Feature Request: snapshot folders

### DISCUSSION THREAD

**Discussion by gitrect** [grafana/grafana#37007] [Dashboards, Panels & Visualizations]
It would be great to be able to organize saved (local) snapshots in folders as can be done with dashboards.

**Comment by bergquist** [comment]
Hm. Could you elaborate on what kind of folders you would need? I'm hesitant since snapshots are usually not long term data like dashboards but a point in time representation of a dashboard. So I dont see the need to structure them that much. 

If we want to provide more structure I think we should re-use the tags from the snapshotted dashboard.

**Comment by gitrect** [comment]
Maybe I am in the minority, but I have a need to archive snapshots of dashboards permanently.  I do performance testing against a content delivery network and need to record and report metrics on a per test, per release basis.  Grafana dashboards are the centerpiece of my reports and presentations. My data source is graphite, so saving Grafana snapshots is much more efficient than trying to archive/backup graphite timeseries data any other way. I currently use long, descriptive titles when saving the snapshots but it would be very nice to be able to organize them into folders.  

Thanks for your quick reply and consideration.  I am a huge fan and evangelist for Grafana in my organization ;-)

**Comment by bergquist** [comment]
That use case is definitely something we want to support. But considering how few times we heard about it we probably dont have time for this in a long time. But we would more than welcome contributions. 

@soulgalore is this workflow something you have considered?

**Comment by jscott0918** [comment]
The ability to tag/store/organize/export snapshots to long term storage would be great. We use grafana for long term storage, then snapshots to archive interesting phenomena or the results of tests. Being able to point them at an s3 bucket/google drive/nfs export/ftp server and label/tag/organize them would be fantastic.

**Comment by vellala2000** [comment]
I am exactly looking for this feature. 

It would be very helpful we can organize snapshots the same way as we do dashboard in grafana. please consider developing this feature.

real time DB query sometimes can take long to load than snapshots. Also dashboard input parameters to retrieve old data can be difficult to find/pin point

**Comment by abadelt** [comment]
Same for me. I'm keeping certain snapshots from performance tests long-term, but the list gets longer and longer. Some means of organization would be very helpful.

**Comment by Lcasaresalva** [comment]
Same here :)

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82997. Feel free to continue the discussion around this there. Thank you!
